openshift-tmp/README.md

```diff
@@ -0,0 +1,5 @@
+# Openshift
```

Comment:
Call this directory openshift itself, instead of openshift-tmp. It is likely here to stay.